docs/build-build-with-polkadot.md

```diff
@@ -19,33 +19,38 @@ that you can get started creating your application in anticipation of the Polkad
 - ink!: **In development** ([Documentation](https://substrate.dev/substrate-contracts-workshop/https://substrate.dev/substrate-contracts-workshop/))
 
-**What you need to know**: Polkadot is currently on the v0.6 release with a running testnet named Alexander and Kusama,
-a value-bearing canary network. The Alexander testnet has been live for over 6 months through the entire PoC-3 and
-PoC-4 release cycles. Polkadot is being built with [implementations in various programming
-languages](learn-implementations) ranging from Rust to JavaScript. Currently the leading implementation is built in
-Rust and using the Substrate framework for developing blockchains. Substrate is a library that allows developers to
-more easily develop entire blockchains by building from a base layer including a networking protocol, consensus, and a
-Wasm interpreter. An in-development extension to Substrate called Cumulus will allow any Substrate built chain to
-connect to Polkadot and become a parachain. Substrate has had a 1.0.0 release, which settled its API for the time
-being.
+## What you need to know
 
-Substrate chains can include smart contract functionality by using the provided [contracts Frame pallet](https://github.com/paritytech/substrate/tree/master/frame/contracts).
-This pallet allows for smart contracts compiled to Wasm code to be deployed to the chain and instantiated by users.
-To facilitate development of Wasm smart contracts, Parity is also developing [ink!](https://github.com/paritytech/ink), a domain specific language built
-in Rust that is intended for writing smart contracts.
+Polkadot is currently on the v0.7 release with a testnet called Westend and a
+value-bearing canary network called Kusama. Polkadot is being built with [implementations in various programming
+languages](learn-implementations) ranging from Rust to JavaScript. Currently the leading implementation is built in
+Rust and built on the Substrate framework. Substrate is a library that allows developers to
+develop entire blockchain applications with ease by bundling together a networking protocol, consensus, and
+Wasm interpreter. Cumulus, an extension to Substrate, will allow any Substrate built chain to
+connect to Polkadot and become a parachain. Substrate is currently nearing its official
+2.0.0 tagged release that will solidify its API.
+
+Polkadot does not natively support smart contracts, however there will be parachains
+that do. Substrate chains can include smart contract functionality by using the provided 
+[contracts Frame pallet](https://github.com/paritytech/substrate/tree/master/frame/contracts).
+The contracts pallet enables a chain to use Wasm-compiled contracts that can be deployed
+permissionlessly by users or with specific rules dependent on the chain. 
+To facilitate development of Wasm smart contracts, Parity is also developing 
+[ink!](https://github.com/paritytech/ink), a domain specific language built
+in Rust for writing smart contracts.
```

Comment:
Let's mention the EVM pallet here as well?

Reply:
Done